src/server_manager/web_app/ui_components/outline-server-view.js

```diff
@@ -628,7 +629,7 @@ export class ServerView extends DirMixin(PolymerElement) {
         serverPortForNewAccessKeys: Number,
         isAccessKeyPortEditable: Boolean,
         serverCreationDate: Date,
-        serverLocation: String,
+        serverLocation: Object,
```

Comment:
Revert to string

Reply:
I think it should acttually be serverLocationId

Reply:
`serverLocation` is a `CloudLocation`.  (Passing just a message-id like `'geo-iowa'` would not be able to display the data center ID in the unknown-location case.)

Reply:
To clarify:
```suggestion
        cloudLocation: Object,
```

Reply:
Done